### PR TITLE
Revert "Use as_column instead of full"

### PR DESCRIPTION
**Pull request by vyasr**
Reverts rapidsai/cudf#14698

That change is causing a number of issues downstream that we don't want to try to resolve right now.

### CODE REVIEW

**Review by jakirkham — APPROVED** [review]

**Review by galipremsagar — APPROVED** [review]

File: python/cudf/cudf/utils/utils.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2024, NVIDIA CORPORATION.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
```
Comment (jakirkham) [inline python/cudf/cudf/utils/utils.py:1]:
```suggestion
# Copyright (c) 2020-2024, NVIDIA CORPORATION.
```

**Review by jakirkham — COMMENTED** [review]
Looks like the style check fails due to copyright year ranges

**Review by mroeschke — APPROVED** [review]

**Comment by bdice** [comment]
Blocking merge of this reversion: https://github.com/rapidsai/cugraph/pull/4215 should fix the last known issue introduced by #14698.

**Review by galipremsagar — CHANGES_REQUESTED** [review]
blocking merge

**Comment by galipremsagar** [comment]
Closing this as all dependent repos have been updated.

**Pull request closed without merging**